disasterinfosite/models.py

```diff
@@ -135,6 +135,10 @@ class SupplyKit(SingletonModel):
         help_text="More information about building your supply kit. Any web address in here gets turned into a link automatically."
     )
 
+    @property
+    def meals(self):
```

Comment:
I'm not sure we are even going to use the supply kit anymore but this is a better way of doing this.

( @carsonmk - we aren't using the supply kit or important links anymore, right? They are not in the designs. )